R packages chapter

danielinteractive opened the pull request:
closes #5

Commit:
don't build anything

Commit:
don't stage .Rbuildignore

Commit:
start with R packages slides

Commit:
Merge branch 'main' into r-packages

Commit:
minor progress with R folder

Commit:
further progress

Commit:
complete content without exercise

Commit:
Merge remote-tracking branch 'origin/main' into r-packages

Commit:
render

Commit:
add exercise

fpahlke commented on slides/02_r_packages.qmd:
```diff
@@ -7,6 +7,310 @@ description: description. Photo CC0 by cottonbro studio on pexels.com.
+
+## `src` Folder
+
+- Contains sources and headers for any compiled code
```
Maybe better because the code is not compiled yet:
- Contains sources and headers for any code that needs compilation

fpahlke commented on slides/02_r_packages.qmd:
```diff
@@ -7,6 +7,310 @@ description: description. Photo CC0 by cottonbro studio on pexels.com.
+## `src` Folder
+
+- Contains sources and headers for any compiled code
+- Should only contain a single language here (e.g. `C++` or `Fortran` but not both)
```
But in principle you can mix different languages, e.g., C and C++

Commit:
Friedrich's comments

danielinteractive commented:
@kkmann is it ok that I merge? or any problems with the rendered files?

kkmann committed:
added description; reduced font size

kkmann committed:
removed _site

kkmann committed:
updated docs folder

kkmann committed:
merge main

Merge branch 'main' into r-packages

# Conflicts:
#	docs/search.json
#	docs/site_libs/revealjs/dist/theme/quarto.css
#	docs/sitemap.xml
#	docs/slides/listing.html
